trl/trainer/orpo_trainer.py

```diff
@@ -659,7 +672,7 @@ def get_batch_logps(
         loss_mask = labels != label_pad_token_id
 
         # dummy token; we'll ignore the losses on these tokens later
-        labels[labels == label_pad_token_id] = 0
+        labels = torch.where(labels == label_pad_token_id, 0, labels)
```

Comment:
Is it a necessary change? Personal opinion, I find it a bit less intuitive to read